dgraph/cmd/zero/zero_test.go

```diff
@@ -112,3 +118,87 @@ func TestProposalKey(t *testing.T) {
 	}
 	require.Equal(t, len(uniqueKeys), 10, "each iteration should create unique key")
 }
+
+func validateJsonHealthContent(body []byte) error {
+	var r map[string]interface{}
+	e := json.Unmarshal(body, &r)
+	if e != nil {
+		return errors.Wrap(e, "json unmarshal error")
+	}
+
+	for k, v := range r {
```

Comment:
I personally would avoid using the switch case here because it doesn't make sure that all the fields are present. I would rather do it brute force and check for each field that I care about. It is okay if the code is a little repeated.